docs/services_03_transactions.md

```diff
@@ -25,8 +25,8 @@ for running the project.
 
 Configure the parameters needed on <span style="color:#DB3A3D">`.env`</span>. These parameters **need to be changed**:
 - <span style="color:#DB3A3D">`ETHEREUM_NODE_URL`</span>: Http/s address of a ethereum node. It can be the same than <span style="color:#DB3A3D">`ETHEREUM_TRACING_NODE_URL`</span>.
-- <span style="color:#DB3A3D">`ETHEREUM_TRACING_NODE_URL`</span>: Http/s address of a Ethereum Parity node with
-[tracing enabled](https://wiki.parity.io/JSONRPC-trace-module).
+- <span style="color:#DB3A3D">`ETHEREUM_TRACING_NODE_URL`</span>: Http/s address of an Openethereum/Parity node with
```

Comment:
Small feedback, but they call it Open**E**thereum I think (https://github.com/openethereum/openethereum)